system-tests/test/deprecated_spec.ts

```diff
@@ -64,6 +64,25 @@ describe('deprecated before:browser:launch args', () => {
     project: beforeBrowserLaunchProject,
     spec: 'app.cy.js,app_spec2.js',
     snapshot: true,
+    browser: ['electron'],
```

Comment:
if the only difference b/t this and the existing test is the snapshots, then you could use `onRun` to change the snapshot name via `originalTitle`. i think this should work:

```suggestion
    onRun: (exec, browserName) => {
      const snapshotSlug = browserName === 'electron' ? 'electron' : 'chromium+firefox';
      return exec({ originalTitle: `concat returns once per spec - ${snapshotSlug}` })
    },
```

Reply:
Good idea. Didn't know about that: https://github.com/cypress-io/cypress/commit/cc71a674b809f9d1441afc72eff754e494502715